### ISSUE TITLE
Logs not showing

### ISSUE THREAD

**Issue by douglascrp** [Alfresco/Aikau#140]
I'm trying to execute the chapter 4.

I've added this block of code into the home.get.js file's services section:

`
    {
      name: "alfresco/services/LoggingService",
      config: {
        loggingPreferences: {
          enabled: true,
          all: true
        }
      }
    },
`
Everything keep working, but I can't see the log message created using this:

`this.alfLog("log", "Setting greeting message!");`

**Comment by douglascrp** [comment]
This is my full services section:

`
services : [ {
		name : "alfresco/services/LoggingService",
		config : {
			loggingPreferences : {
				enabled : true,
				all : true
			}
		}
	}, "alfresco/services/NavigationService",
			"alfresco/services/LogoutService",
			"alfresco/services/DocumentService",
			"alfresco/services/ActionService",
			"alfresco/services/UploadService" ],
`

Is this right?

**Comment by draperd** [comment]
Yes, that does look correct - and you're indicating that everything else is working. Are you seeing any other logs being displayed or is it just the log form the HelloWorld widget?

**Comment by draperd** [comment]
Are you still having problems with this? Otherwise I'll close

**Comment by douglascrp** [comment]
@draperd Yes, it's still happening, but I couldn't try to fix it again since that day.
I'll give it a try today in the afternoon and will get feedback on it.

Thanks

**Comment by draperd** [comment]
Still not heard anything on this... I'm going to close for now, but please re-open if you get a chance to try it out again and you're still having issues.

**Comment by douglascrp** [comment]
@draperd Just to give you the feedback I promissed long time ago... I'm trying the tutorial again, and this time log worked.

**Comment by douglascrp** [comment]
@draperd Sorry, but I misinterpreted the result of my new test.
What I see in the console are the Alfresco's log messages, but not my custom Header widget's message.
So, I still have the same problem.

**Comment by douglascrp** [comment]
@draperd Me again.
I really can't understand what is wrong with my project, but I downloaded your source code and replaced mine with yours, tried it and the log appeared.
Then I replaced again your code with mine, and now my code works.
I even changed the home.get.js file in order to make sure my code is load by replacing the logo position to right, left, and so on, and for sure, that is my code running.
And the log is still working.

I really don't know what fixed it.
